reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java

```diff
@@ -48,20 +52,22 @@
     @Path("/scm")
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "Get dependency report for a project specified in a repository url")
+    @ApiOperation(value = "Get dependency report for a project specified in a repository URL",
+            hidden = true)
+    // TODO unhide when the method will be implemented
     public Report scmGenerator(SCMRequest scmRequest) {
-        // TODO Auto-generated method stub
-        return null;
+        throw new NotImplementedException();
     }
 
     @POST
     @Path("/gav")
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "Get dependency report for a GAV")
-    public Report gavGenerator(GAVRequest gavRequest) {
-        // TODO Auto-generated method stub
-        return null;
+    @ApiOperation(
+            value = "Get dependency report for a GAV " // TODO change when dependencies will be implemented
+                    + "(Currently the dependencies and dependency_versions_satisfied don't contains usefull values)")
+    public Report gavGenerator(GAV gavRequest) {
```

Comment:
Currently we do not support products in requests so I removed it. I'm not sure if it is the best approach